runtime/runtime/src/store.rs

```diff
@@ -16,8 +15,7 @@ pub enum StateRecord {
     Contract { account_id: AccountId, code: String },
     /// Access key associated with some account.
     AccessKey { account_id: AccountId, public_key: ReadablePublicKey, access_key: AccessKey },
-    /// Callback.
-    Callback { id: Vec<u8>, callback: Callback },
+    // TODO: DATA
```

Comment:
we need it for state dump to work, right?

Reply:
Yes. Will push it separately